beacon-chain/core/epoch/precompute/justification_finalization.go

```diff
@@ -100,6 +100,8 @@ func weighJustificationAndFinalization(state state.BeaconState, newBits bitfield
 	return state, nil
 }
 
+var ComputeCheckpoints = computeCheckpoints // For test only.
```

Comment:
this should be in a test file like justification_test.go, with `package precompute` at the top